steal.lic

```diff
@@ -287,7 +287,7 @@ class Steal
   end
 
   def put_item?(item)
-    case bput("put my #{item} in my #{@stealing_bag}", 'You put', 'no matter how you arrange it', 'The .* is too \w+ to fit in', 'There isn\'t any more room', 'perhaps try doing that again')
+    case bput("put my #{item} in my #{@stealing_bag}", 'You put', 'no matter how you arrange it', 'The .* is *.* too \w+ to fit in', 'There isn\'t any more room', 'perhaps try doing that again')
```

Comment:
What does this match?

Reply:
`The thing is just too something to fit in`

Reply:
Figured it could be others as well, not just *just*.